### PR TITLE
Stop rails engine

### PR DESCRIPTION
**Pull request by myokoym**
ref: #2

### CODE REVIEW

**Comment by mtsmfm** [comment]
LGTM

**Comment by mtsmfm** [comment]
https://daimon-news-multi-tenant-pr-8.herokuapp.com/sites/2/posts

File: app/views/layouts/site_posts.html.erb [hunk 1]
```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title><%= @site.name %></title>
+  <%= stylesheet_link_tag    'application', media: 'all' %>
+  <%= javascript_include_tag 'application' %>
+
+  <%# TODO: Use Asset Pipeline %>
+  <link rel="stylesheet" type="text/css" href="/site<%= @site.id %>/site.css" media="all">
+  <script src="/site<%= @site.id %>/site.js"></script>
```
Comment (mtsmfm) [inline app/views/layouts/site_posts.html.erb]: https://github.com/bm-sms/daimon-news-multi-tenant/commit/ad482f822a5e1e60da0f023f9162b14926637d1a#diff-8d9733948ac2a35133e5677b89969197R10

**Comment by tricknotes** [comment]
:dizzy:

**Comment by myokoym** [comment]
:smile:

**Pull request merged**